guides/common/attributes-titles.adoc

```diff
@@ -50,9 +50,11 @@ ifdef::satellite[]
 :InstallingSmartProxyDocTitle: Installing Capsule Server
 :ManagingConfigurationsPuppetDocTitle: Managing Configurations Using Puppet Integration in {ProjectName}
 :PlanningDocTitle: {Project} Overview, Concepts, and Deployment Considerations
+:QuickstartDocTitle: Quickstart Guide
 endif::[]
 
 ifdef::orcharhino[]
 :InstallingServerDocTitle: Installing {Project} Server
 :InstallingSmartProxyDocTitle: Installing {Project} Proxy
+:QuickstartDocTitle: Quickstart Guide
```

Comment:
@maximiliankolb Same as above, can you suggest something for orcharhino?

Reply:
We don't use this guide/title right now. So whatever you deem fine for the community works for me too.

Reply:
@evgeni I would remove the title for orcharhino in this case.

Reply:
I think the best option is to keep it in sync with the title for "foreman-el".